client-libs/typescript/src/openai-legacy/index.ts

```diff
@@ -0,0 +1,90 @@
+// import * as openPipeClient from "../codegen";
```

Comment:
Commented this out because I was getting typescript errors, will need to re-enable it once we get to trying to implement.